lib/api/v3/work_packages/work_package_contract.rb

```diff
@@ -34,20 +34,22 @@ module API
   module V3
     module WorkPackages
       class WorkPackageContract < Reform::Contract
-        WRITEABLE_ATTRIBUTES = [
-          'lock_version',
-          'subject',
-          'parent_id',
-          'description',
-          'start_date',
-          'due_date',
-          'status_id',
-          'assigned_to_id',
-          'responsible_id',
-          'priority_id',
-          'category_id',
-          'fixed_version_id'
-        ].freeze
+        def self.writable_attributes
+          @writable_attributes ||= [
+              'lock_version',
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.